### PR TITLE
[auto-fix] interface type updated for Osmosis1TrxMsgIbcApplicationsTransferV1MsgTransfer

### PR DESCRIPTION
**Pull request by gurbaj5124871**
**This is an automated generated pr**
**changelog**
- auto-fix: interface type updated for Osmosis1TrxMsgIbcApplicationsTransferV1MsgTransfer
    
**Block Data**
network: osmosis-1
height: 12427622
data:
```
[
  {
    "sourcePort": "transfer",
    "sourceChannel": "channel-208",
    "token": {
      "denom": "ibc/903A61A498756EA560B85A85132D3AEE21B5DEDD41213725D22ABF276EA6945E",
      "amount": "38997522"
    },
    "sender": "osmo1cazym65024jv8pk0a4xnkpn7hu0g3hrgafw8zp",
    "receiver": "axelar1cazym65024jv8pk0a4xnkpn7hu0g3hrg3utllj",
    "timeoutHeight": {
      "revisionNumber": "1",
      "revisionHeight": "10134217"
    }
  },
  {
    "sourcePort": "transfer",
    "sourceChannel": "channel-1429",
    "token": {
      "denom": "ibc/23AB778D694C1ECFC59B91D8C399C115CC53B0BD1C61020D8E19519F002BDD85",
      "amount": "24462531662828528515"
    },
    "sender": "osmo1uvkwef43394s0g4gd9lg3uyqjvyphttxxedfj5",
    "receiver": "archway1uvkwef43394s0g4gd9lg3uyqjvyphttxmfzaw3",
    "timeoutHeight": {
      "revisionNumber": "1",
      "revisionHeight": "2031441"
    }
  }
]
```

### CODE REVIEW

File: src/types/chain/osmosis-1/IRangeBlockOsmosis1TrxMsg.ts
Comment (gurbaj5124871) [inline src/types/chain/osmosis-1/IRangeBlockOsmosis1TrxMsg.ts:1]: **This is an automated generated comment**
**Block Data**
network: osmosis-1
height: 12434821
data:
```
[
  {
    "sourcePort": "transfer",
    "sourceChannel": "channel-6994",
    "token": {
      "denom": "ibc/D79E7D83AB399BFFF93433E54FAA480C191248FC556924A2A8351AE2638B3877",
      "amount": "159700000"
    },
    "sender": "osmo1pma6x4wxru55000ysdlj70zw46dgywyd0qjny0",
    "receiver": "celestia1pma6x4wxru55000ysdlj70zw46dgywydk3sngs",
    "timeoutHeight": {
      "revisionHeight": "151348"
    }
  }
]
```

errors:
```
[
  {
    "path": "$input.transactions[2].messages[0].data.timeoutTimestamp",
    "expected": "string"
  }
]
```

interface:
```
export interface Osmosis1TrxMsgIbcApplicationsTransferV1MsgTransfer {
    type: string;
    data: Osmosis1TrxMsgIbcApplicationsTransferV1MsgTransferData;
}
interface Osmosis1TrxMsgIbcApplicationsTransferV1MsgTransferData {
    sourcePort: string;
    sourceChannel: string;
    token: Osmosis1TrxMsgIbcApplicationsTransferV1MsgTransferToken;
    sender: string;
    receiver: string;
    timeoutHeight: Osmosis1TrxMsgIbcApplicationsTransferV1MsgTransferTimeoutHeight;
}
interface Osmosis1TrxMsgIbcApplicationsTransferV1MsgTransferToken {
    denom: string;
    amount: string;
}
interface Osmosis1TrxMsgIbcApplicationsTransferV1MsgTransferTimeoutHeight {
    revisionHeight: string;
}

```

Comment (gurbaj5124871) [inline src/types/chain/osmosis-1/IRangeBlockOsmosis1TrxMsg.ts:1]: **This is an automated generated comment**
**Block Data**
network: osmosis-1
height: 12434821
data:
```
[
  {
    "sourcePort": "transfer",
    "sourceChannel": "channel-6994",
    "token": {
      "denom": "ibc/D79E7D83AB399BFFF93433E54FAA480C191248FC556924A2A8351AE2638B3877",
      "amount": "159700000"
    },
    "sender": "osmo1pma6x4wxru55000ysdlj70zw46dgywyd0qjny0",
    "receiver": "celestia1pma6x4wxru55000ysdlj70zw46dgywydk3sngs",
    "timeoutHeight": {
      "revisionHeight": "151348"
    }
  }
]
```

errors:
```
[
  {
    "path": "$input.transactions[2].messages[0].data.timeoutTimestamp",
    "expected": "string"
  }
]
```

interface:
```
export interface Osmosis1TrxMsgIbcApplicationsTransferV1MsgTransfer {
    type: string;
    data: Osmosis1TrxMsgIbcApplicationsTransferV1MsgTransferData;
}
interface Osmosis1TrxMsgIbcApplicationsTransferV1MsgTransferData {
    sourcePort: string;
    sourceChannel: string;
    token: Osmosis1TrxMsgIbcApplicationsTransferV1MsgTransferToken;
    sender: string;
    receiver: string;
    timeoutHeight: Osmosis1TrxMsgIbcApplicationsTransferV1MsgTransferTimeoutHeight;
}
interface Osmosis1TrxMsgIbcApplicationsTransferV1MsgTransferToken {
    denom: string;
    amount: string;
}
interface Osmosis1TrxMsgIbcApplicationsTransferV1MsgTransferTimeoutHeight {
    revisionHeight: string;
}

```

Comment (gurbaj5124871) [inline src/types/chain/osmosis-1/IRangeBlockOsmosis1TrxMsg.ts:1]: **This is an automated generated comment**
**Block Data**
network: osmosis-1
height: 12434867
data:
```
[
  {
    "sourcePort": "transfer",
    "sourceChannel": "channel-0",
    "token": {
      "denom": "ibc/27394FB092D2ECCD56123C74F36E4C1F926001CEADA9CA97EA622B25F41E5EB2",
      "amount": "222868600"
    },
    "sender": "osmo1t6wjr00y84n9wfmk8st8e3h8xzuzegl39sjnyc",
    "receiver": "cosmos1t6wjr00y84n9wfmk8st8e3h8xzuzegl3dtprj2",
    "timeoutHeight": {
      "revisionNumber": "4",
      "revisionHeight": "17941780"
    }
  }
]
```

errors:
```
[
  {
    "path": "$input.transactions[9].messages[0].data.timeoutTimestamp",
    "expected": "string"
  }
]
```

interface:
```
export interface Osmosis1TrxMsgIbcApplicationsTransferV1MsgTransfer {
    type: string;
    data: Osmosis1TrxMsgIbcApplicationsTransferV1MsgTransferData;
}
interface Osmosis1TrxMsgIbcApplicationsTransferV1MsgTransferData {
    sourcePort: string;
    sourceChannel: string;
    token: Osmosis1TrxMsgIbcApplicationsTransferV1MsgTransferToken;
    sender: string;
    receiver: string;
    timeoutHeight: Osmosis1TrxMsgIbcApplicationsTransferV1MsgTransferTimeoutHeight;
}
interface Osmosis1TrxMsgIbcApplicationsTransferV1MsgTransferToken {
    denom: string;
    amount: string;
}
interface Osmosis1TrxMsgIbcApplicationsTransferV1MsgTransferTimeoutHeight {
    revisionNumber: string;
    revisionHeight: string;
}

```

Comment (ckuroki) [reply src/types/chain/osmosis-1/IRangeBlockOsmosis1TrxMsg.ts:1]: `timeoutTimestamp` data type is unsigned int64

https://github.com/cosmos/ibc-go/blob/947b15ed0e3d2130e5bc9763ce623f6cbb7aa74a/proto/ibc/core/channel/v1/channel.proto#L124

**Pull request merged**